spec/services/catalog/import_service_plans_spec.rb

```diff
@@ -30,6 +30,24 @@
       end
     end
 
+    context "when force_reset is set" do
+      let!(:service_plan_current) { create(:service_plan, :portfolio_item => portfolio_item) }
+      let(:data) { [service_plan] }
+
+      before do
+        described_class.new(portfolio_item.id, true).process
+      end
+
+      it "destroys the current service_plans" do
```

Comment:
Since this is supposed to destroy multiple, should we adjust the spec to ensure that multiple get destroyed?

Reply:
The following spec shows that is happening in my opinion:
You end up taking both as completing the behavior: clear everything and reimport
```
 it "reimports the service plan" do
   portfolio_item.service_plans.reload
   expect(portfolio_item.service_plans.any?).to be_truthy
end
```